go.mod

```diff
@@ -0,0 +1,37 @@
+module github.com/giantswarm/certs
+
+go 1.12
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/giantswarm/microerror v0.0.0-20181001144842-3bc3cb1a3670
+	github.com/giantswarm/micrologger v0.0.0-20190118112544-0926d9b7c541
+	github.com/go-kit/kit v0.6.0 // indirect
+	github.com/go-logfmt/logfmt v0.4.0 // indirect
+	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/gogo/protobuf v1.2.0 // indirect
+	github.com/google/btree v1.0.0 // indirect
+	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
+	github.com/googleapis/gnostic v0.2.0 // indirect
+	github.com/gregjones/httpcache v0.0.0-20181110185634-c63ab54fda8f // indirect
+	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/juju/errgo v0.0.0-20140925100237-08cceb5d0b53 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/stretchr/testify v1.3.0 // indirect
+	golang.org/x/crypto v0.0.0-20190131182504-b8fe1690c613 // indirect
+	golang.org/x/net v0.0.0-20190125091013-d26f9f9a57f3 // indirect
+	golang.org/x/oauth2 v0.0.0-20190130055435-99b60b757ec1 // indirect
+	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4
+	golang.org/x/sys v0.0.0-20190201152629-afcc84fd7533 // indirect
+	golang.org/x/time v0.0.0-20181108054448-85acf8d2951c // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+	k8s.io/api v0.0.0-20190222213804-5cb15d344471
```

Comment:
It seems as if k8s doesn't support stable semvers for go modules yet? Only `client-go` has proper tags (from what I can see / get working).

Nevertheless, these `v0.0.0...` commits are results from `go get k8s.io/api@kubernetes-v1.13.5` so maybe they are still fine? According to vgo presentation by rsc and how `v0.0.0` is special, this still makes me a bit uncomfortable..

Reply:
I think we will have this for some time. It is what it is for now. I am just wondering if we can drop the vendor dir now.

Reply:
That was my main reason I didn't start with go modules earlier.

In the future I think we'll be able to drop them, we'll have to wait till this stuff is released: https://github.com/kubernetes/client-go/blob/89226ff90625425e6c7a87fe219dcefd75d28e53/go.mod#L30

I'd prefer to use semver version of client-go already. I think we should also add some TODOs/issues to track that.

Reply:
Regarding `client-go`, as noted in description / go.mod, I already used semver for that.

@xh3b4sd Would it make sense to track this TODO-issue on Team #2 board as that's focused towards k8s releases?

Reply:
I am not sure this is specific to any team. Is this blocking us from going with it or is it only not cool?

Reply:
I think this may have a nasty implication as having old version of apiextensions because go modules take the lowest compatible version AIUI.

Reply:
Let's discuss this in sync tomorrow.